plugins/topology/src/hooks/useForceUpdate.ts

```diff
@@ -0,0 +1,9 @@
+import * as React from 'react';
+
+export const useForceUpdate = () => {
+  const [, setTick] = React.useState(0);
```

Comment:
```suggestion
  const [ setTick] = React.useState(0);
```

Reply:
am not sure if the above suggestion will work as `useState` returns a tuple with the first position will return an item and the second setter for it